.circleci/ci/accessibility-axe.js

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
```

Comment:
If I merge this PR, all the PR builds would fail, since accessibility check is failing. Can you unblock this use-case ?

Reply:
@rismehta  for that I have added Ids for which accessibility check was failing in the collateral as *whitelistedLabel* array, we are checking this array, when deciding to fail the pipeline.
This would not impact the report for accessibility, which will contain all the violations.

Reply:
ci/circleci: cypress-chrome-cloudready-with-latest-addon — Your tests passed on CircleCI!

This passed, but with addon got failed in UI tests stage, have re-run it, will update once this build is success